docs: add a security policy (#586)
abiosoft/colima: SECURITY.md

Thanks for helping make Colima safe for everyone.

## Security

We take the security of Colima seriously.

We will ensure that your finding gets passed along to the appropriate maintainers for remediation. 

## Reporting Security Issues

> If you believe you have found a security vulnerability in any GitHub-owned repository, please report it to us through coordinated disclosure.

Author: ```suggestion
If you believe you have found a security vulnerability in this repository, please report it to us through coordinated disclosure.
```
